.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-Drevision=HEAD-SNAPSHOT
```

Comment:
The idea is, that `gwt-event` has no parent pom - cause we only want to deploy this module - and the parent pom needs not to be deployed. Atm, it is, but I think it is not necessary and we should stop this.

Reply:
We probably do want an in-project parent for this repo, @FrankHossfeld, since there is more than one module present here. The other projects like gwt-timer, gwt-http etc are exceptions in this way.

Reply:
flatten-maven-plugin makes it possible.
I use it here: https://github.com/tbroyer/jaxrs-utils and https://github.com/tbroyer/gwt-maven-archetypes
Configuration for all modules is done in the root POM and inherited in the actual modules, which won't reference the parent POM thanks to the flatten-maven-plugin (see https://search.maven.org/artifact/net.ltgt.jaxrs/resteasy-client-okhttp3/1.1/jar for example).
In these projects, I simply avoid deploying the root project (by using `-pl <submodule>` while omitting `-am`) because each submodule has its own release lifecycle, but this could be done by configuring the root POM to skip deployment and reconfiguring each submodule to actually deploy.

Reply:
please review again.

Reply:
Done